ffi/rust/README.md

```diff
@@ -0,0 +1,55 @@
+# Iotics Identity - Foreign Function Interface
+A Rust cargo for utilising a Go library for Decentralised Identity management through FFI.
+
+
+## Prerequisites
+Install Rust - [official instructions](https://www.rust-lang.org/tools/install) - i.e.:
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source $HOME/.cargo/env
+```
+Install Rust crate (for creating C bindings):
+```shell
+cargo install bindgen
```

Comment:
Is this needed? Can we just add it as a dependency inside Cargo.toml and build.rs can use it internally?